dashboard/modules/actor/actor_head.py

```diff
@@ -230,6 +230,11 @@ async def kill_actor(self, req) -> aiohttp.web.Response:
 
         return rest_response(success=True, message=f"Killed actor with id {actor_id}")
 
+    @routes.get("/api/v0/actors")
+    async def get_actors(self, req) -> aiohttp.web.Response:
```

Comment:
Could we add some comments for each rest api: e.g. what the api is returning.

Reply:
I am planning to have a follow up PR to improve docstring + API output! I will address it in that PR!